src/TestFramework.Extensions.UAP/TestFramework.Extensions.UAP.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>uap10.0</TargetFrameworks>
+    <RootNamespace>Microsoft.VisualStudio.TestTools.UnitTesting</RootNamespace>
+    <AssemblyName>Microsoft.VisualStudio.TestPlatform.TestFramework.Extensions.UAP</AssemblyName>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>MSTest.TestFramework.Extensions.UWP</PackageId>
```

Comment:
It's weird to have dll named `UAP` and package named `UWP`.

Reply:
Haha I know. I'll fix it.